### PR TITLE
HPC: Add a list of HPC software

### PR DESCRIPTION
**Pull request by lrbison**
Add a list of commercial software vendors and applications which support Graviton instances.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: HPC/README.md [hunk 1]
```diff
@@ -485,3 +485,15 @@ GCC	| 11	| -O3 -mcpu=neoverse-v1	| -fopenmp	| -ffast-math
 CLang/LLVM |	14 | -O3 -mcpu=neoverse-512tvb	| -fopenmp	| -ffast-math
 Arm Compiler for Linux |	23.04 |	-O3 -mcpu=neoverse-512tvb |	-fopenmp |	-ffast-math
 Nvidia HPC SDK |	23.1	| -O3 -tp=neoverse-v1	| -mp |	-fast
+
+### Commercial HPC applications list for Graviton
+Below is a list of some of the commercial HPC applications that are available on Graviton.
+ISV                   | Application   | Release of support  | Additional Notes
+----------------------|---------------|---------------------|-----------------
+Ansys                 | Fluent        | v221                | [AWS Blog](https://aws.amazon.com/blogs/compute/running-ansys-fluent-on-amazon-ec2-c5n-with-elastic-fabric-adapter-efa)
+Ansys                 | LS-Dyna       | 12.1                | [ISV Article](https://www.ansys.com/content/dam/amp/2023/june/quick-request/)
```
Comment (AWSjswinney) [inline HPC/README.md:494]: 404 😭

File: HPC/README.md [hunk 2]
```diff
@@ -485,3 +485,15 @@ GCC	| 11	| -O3 -mcpu=neoverse-v1	| -fopenmp	| -ffast-math
...
+Ansys                 | RedHawk-SC    | 2023R1              | [Release Notes](https://www.ansys.com/content/dam/it-solutions/platform-support/arm-64-processor-support-announcement-january-2023.pdf)
+Fritz Haber Institute | FHIaims       | 21.02               | [AWS Blog](https://aws.amazon.com/blogs/hpc/quantum-chemistry-calculation-on-aws/)
+Mathworks             | Matlab        | R2023b              | [Release Notes](https://www.mathworks.com/products/new_products/release-highlights.html), [ISV Article](https://www.mathworks.com/support/requirements/apple-silicon.html)
+Sentieon              | DNAseq , TNseq, DNAscope | 202112.02 | [Release Notes](https://support.sentieon.com/manual/appendix/releasenotes/#release-202112-02), [AWS Blog](https://aws.amazon.com/blogs/hpc/cost-effective-and-accurate-genomics-analysis-with-sentieon-on-aws/)
+Siemens               | StarCCM++     | 2023.2              | [Release Notes](https://blogs.sw.siemens.com/simcenter/simcenter-star-ccm-2302-released/arm-64-processor-support-announcement-june-2023.pdf)
```
Comment (AWSjswinney) [inline HPC/README.md:499]: 404 here as well.

**Review by AWSjswinney — CHANGES_REQUESTED** [review]
oops, some links are broken!

File: HPC/README.md (hunk 2 above)
Comment (AWSjswinney) [reply HPC/README.md:507]: Does this link to the Siemens website require a login of some kind? I'm still getting a 404.

Comment (lrbison) [reply HPC/README.md]: Yes, I noticed that right before lunch.  Should all be fixed now.

**Review by AWSjswinney — APPROVED** [review]

File: HPC/README.md [hunk 3]
```diff
@@ -492,3 +492,17 @@ GCC	| 11	| -O3 -mcpu=neoverse-v1	| -fopenmp	| -ffast-math
 CLang/LLVM |	14 | -O3 -mcpu=neoverse-512tvb	| -fopenmp	| -ffast-math
 Arm Compiler for Linux |	23.04 |	-O3 -mcpu=neoverse-512tvb |	-fopenmp |	-ffast-math
 Nvidia HPC SDK |	23.1	| -O3 -tp=neoverse-v1	| -mp |	-fast
+
+### Common HPC Applications on Graviton
+Below is a list of some common HPC applications that run on Graviton.
+ISV                   | Application   | Release of support  | Additional Notes
+----------------------|---------------|---------------------|-----------------
+Ansys                 | Fluent        | v221                | [Graviton Applications (AWS)](https://aws.amazon.com/blogs/hpc/application-deep-dive-into-the-graviton3e-based-amazon-ec2-hpc7g-instance/)
+Ansys                 | LS-Dyna       | 12.1                | [Graviton Applications (AWS)](https://aws.amazon.com/blogs/hpc/application-deep-dive-into-the-graviton3e-based-amazon-ec2-hpc7g-instance/)
```
Comment (juntangc) [inline HPC/README.md:501]: we can also refer to this link 
https://rescale.com/blog/rescale-automates-the-deployment-of-ansys-ls-dyna-and-ansys-fluent-workloads-on-amazon-ec2-hpc7g-instances/

Comment (lrbison) [reply HPC/README.md]: Added.  Thank you.

**Review by juntangc — APPROVED** [review]

**Pull request merged**